src/03-pages/lab/lab/lab-page.tsx

```
@@ -102,30 +105,38 @@ const LabPage = () => {
   /**
-   * 브라우저 환경 및 화면 너비를 감지하여 모바일 모드 여부 결정함
-   * Window Resize 이벤트를 구독하여 실시간 대응함
+   * 라우팅 판정. 최초 진입 시 UA 로 1회만 결정함
+   *
+   * 화면 너비를 여기 섞으면 운영자가 창을 좁히는 것만으로 측정 도중 대시보드가
+   * 언마운트되어 소켓 구독과 페어링 상태가 소실됨 (FE #78). 너비는 아래 배너
+   * 표시 전용으로 분리함
    */
   useEffect(() => {
     if (!isClient) return;
+    setIsMobileUA(/Android|iPhone|iPad|iPod/i.test(navigator.userAgent));
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/src/03-pages/lab/lab/lab-page.tsx:116:5
  114 |   useEffect(() => {
  115 |     if (!isClient) return;
> 116 |     setIsMobileUA(/Android|iPhone|iPad|iPod/i.test(navigator.userAgent));
      |     ^^^^^^^^^^^^^ Avoid calling setState() directly within an effect
  117 |   }, [isClient]);
  118 |
  119 |   /**

(react-hooks/set-state-in-effect)